kvbc/include/kv_types.hpp

```diff
@@ -10,32 +10,42 @@
 // file.
 //
 
-#ifndef CONCORD_BFT_UTIL_KV_TYPES_H_
-#define CONCORD_BFT_UTIL_KV_TYPES_H_
+#pragma once
 
+#include <string_view>
 #include <unordered_map>
 #include <map>
 #include <vector>
+#include <functional>
 #include "sliver.hpp"
 
-namespace concordUtils {
+namespace concord::kvbc {
 
-typedef Sliver Key;
-typedef Sliver Value;
+typedef concordUtils::Sliver Key;
+typedef concordUtils::Sliver Value;
 typedef std::pair<Key, Value> KeyValuePair;
 typedef std::unordered_map<Key, Value> SetOfKeyValuePairs;
 typedef std::map<Key, Value> OrderedSetOfKeyValuePairs;
 typedef std::vector<Key> KeysVector;
 typedef KeysVector ValuesVector;
-typedef uint64_t BlockId;
+typedef std::uint64_t BlockId;
 
-OrderedSetOfKeyValuePairs order(const SetOfKeyValuePairs &unordered);
-}  // namespace concordUtils
+template <typename ContainerIn, typename ContainerOut>
+ContainerOut order(const ContainerIn& unordered) {
+  ContainerOut out;
+  for (auto&& kv : unordered) out.insert(kv);
+  return out;
+}
```

Comment:
I think it would be better to not allow users to specify types in order to avoid cases such as:

```
order<OrderedSetOfKeyValuePairs, UnorderedSetOfKeyValuePairs>()
```

In that case, this function will not order.

Reply:
agree